subprocess-example.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+import sys
+import boto3
+import datetime
+from shlex import split
+ 
+s3 = boto3.client('s3')
+ 
+def main(argv):
+    cmd = argv
+    log_file_name = datetime.datetime.utcnow().strftime("%m_%d_%Y") + "_logfile"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. The recommended way to create an aware datetime object representing a specific timestamp in UTC is by using `now` or `fromtimestamp` methods and passing `tzinfo` to them.

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)